gravity-forms/gw-prevent-duplicate-selections.php

```diff
@@ -47,7 +47,7 @@ public function output_script() {
 
 		<script type="text/javascript" defer>
 
-			jQuery( function( $ ) {
+			function process_duplicate_selections($) {
```

Comment:
```suggestion
			function gwProcessDuplicateSelections($) {
```

We typically use camel case for JS functions. I also prefixed this with `gw` since I believe this will be in the global scope?